x-pack/plugins/spaces/server/spaces_client/spaces_client.test.ts

```diff
@@ -27,23 +27,23 @@ describe('#getAll', () => {
       attributes: {
         name: 'foo-name',
         description: 'foo-description',
-        bar: 'foo-bar',
+        // bar: 'foo-bar', // what is this for?
```

Comment:
I see that `transformSavedObjectToSpace` used to `...savedObject.attributes as Space`. Maybe these made-up fields were to make sure devs knew that all fields would be copied over, since `as Space` wouldn't matter at runtime?